### ISSUE TITLE
Fix paths of binary v4l2rtspserver

### ISSUE THREAD

**Issue by p4u** [Dafang-Hacks/rootfs#4]
Paths of v4lrtspserver must be adapted to the new root fs

```
strings v4l2rtspserver-master | grep detection

/system/sdcard/scripts/detectionTracking.sh
/system/sdcard/scripts/detectionOn.sh
/system/sdcard/scripts/detectionOff.sh
```

That should be done during compilation (no idea how this binary is compiled).

A quick patch would be to change the strings using "sed", but the string size must be the same (or the binary will fail).

**Comment by n1zzo** [comment]
Also `autonight` has wrong hardcoded paths in it.
I think that this rootfs is compiled from https://github.com/Dafang-Hacks/Main.
I'm wondering how to distinguish between the binary that will go into firmware_mod and the one that will go into rootfs.

**Comment by x86dev** [comment]
Yes, I also can confirm this -- there definitely are hardcoded paths in it. Symlinking to other (old) paths is fixing it temporarily ...
